### PR TITLE
glibc: cherry-pick fix for CVE-2023-4911 "Looney Tunables"

### PR DESCRIPTION
**Pull request by baloo**

### CODE REVIEW

**Comment by baloo** [comment]
cc @jsoo1

**Comment by baloo** [comment]
wait nevermind, it doesn't apply cleanly

File: pkgs/development/libraries/glibc/cve-2023-4911.patch
Comment (jsoo1) [inline pkgs/development/libraries/glibc/cve-2023-4911.patch:1]: Hm. I think we need the plaintext for this.

**Review by jsoo1 — CHANGES_REQUESTED** [review]

File: pkgs/development/libraries/glibc/cve-2023-4911.patch
Comment (baloo) [reply pkgs/development/libraries/glibc/cve-2023-4911.patch:1]: yeah, shell is difficult!! (forgot to escape the url in the curl command)

Comment (jsoo1) [reply pkgs/development/libraries/glibc/cve-2023-4911.patch:1]: > yeah, shell is difficult!!
I can commiserate!!!

**Review by jsoo1 — APPROVED** [review]

**Pull request merged**